lib/push.js

```diff
@@ -8,13 +8,16 @@ var P = require('./promise')
 var ERR_NO_PUSH_CALLBACK = 'No Push Callback'
 var ERR_DATA_BUT_NO_KEYS = 'Data payload present but missing key(s)'
 
-var LOG_OP_PUSH_TO_DEVICES = 'push.pushToDevices'
+var LOG_OP_PUSH_TO_DEVICES = 'push.push'
```

Comment:
this will probably get confusing, we have a few events such as `signup.signup` and it takes time to explain the names. Can we do better here?